packages/edit-navigation/src/index.js

```diff
@@ -66,7 +66,7 @@ const fetchLinkSuggestions = (
 					type: 'post',
 					subtype,
 				} ),
-			} )
+			} ).catch( () => [] ) // fail by returning no results
```

Comment:
Is the error important for us to know about?

In other words, will discarding the error entirely be problematic?

Reply:
Maybe. But the failure wasn't handled before this change anyways, so there is no regression.